libs/consts/src/character.ts

```diff
@@ -188,6 +188,10 @@ export type LocationCharacterKey = (typeof allLocationCharacterKeys)[number]
 
 export type LocationKey = LocationCharacterKey | ''
 
+// GO currently only support 4-5 star characters
```

Comment:
Implying its a design decision on our part to only have 4-5 star characters XD

Reply:
I changed it to 

```
// Genshin Impact currently only has 4-5 star characters
```

Is that better? XD